src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -706,9 +755,46 @@ private Map<String, Object> getSolicitorDetails(String authorisation, Map<String
         } catch (Exception e) {
             log.error("Error while fetching User or Org details for the logged in user ", e);
         }
+
         return caseDataUpdated;
 
     }
 
+
+    @PostMapping(path = "/transfer-court/transfer-court-confirmation",
+        consumes = APPLICATION_JSON, produces = APPLICATION_JSON)
+    @Operation(description = "Callback to create confirmation of transfer court ")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Callback processed.",
+            content = @Content(mediaType = "application/json",
+                schema = @Schema(implementation = uk.gov.hmcts.reform.ccd.client.model.CallbackResponse.class))),
+        @ApiResponse(responseCode = "400", description = "Bad Request", content = @Content)})
+    public ResponseEntity<SubmittedCallbackResponse> transferCourtConfirmation(
+        @RequestHeader(HttpHeaders.AUTHORIZATION) @Parameter(hidden = true) String authorisation,
+        @RequestBody CallbackRequest callbackRequest
+    ) {
+        CaseData caseData = CaseUtils.getCaseData(callbackRequest.getCaseDetails(), objectMapper);
+        allTabsService.updateAllTabs(caseData);
```

Comment:
Do we need to update all tabs here?

Reply:
@shashisk97 logic to populate courtname into tab  resides in this method that is why it's  getting used

Reply:
cant we just update the court object alone? the above will update all tabs when there is no change needed

Reply:
This change will need full retesting. Hence, we will pick it up in IP sprint as part of https://tools.hmcts.net/jira/browse/PRL-4334